src/Momento.Sdk/SimpleCacheClient.cs

```diff
@@ -16,17 +17,21 @@ public class SimpleCacheClient : ISimpleCacheClient
 {
     private readonly ScsControlClient controlClient;
     private readonly ScsDataClient dataClient;
+    protected readonly IConfiguration config;
 
     /// <summary>
     /// Client to perform operations against the Simple Cache Service.
     /// </summary>
     /// <param name="authToken">Momento JWT.</param>
     /// <param name="defaultTtlSeconds">Default time to live for the item in cache.</param>
     /// <param name="dataClientOperationTimeoutMilliseconds">Deadline (timeout) for communicating to the server. Defaults to 5 seconds.</param>
-    public SimpleCacheClient(string authToken, uint defaultTtlSeconds, uint? dataClientOperationTimeoutMilliseconds = null)
+    /// <param name="config">Configuration to use for the transport, retries, middlewares. Defaults to <see cref="PrebuiltConfigurations.DevConfig"/></param>
+    public SimpleCacheClient(string authToken, uint defaultTtlSeconds, uint? dataClientOperationTimeoutMilliseconds = null, IConfiguration? config = null)
```

Comment:
> I think we decided that we were going to make the config a required argument. And I would put it as either the first or second argument in the list.
I think making this required will introduce more friction to getting started and cognitive burden. Let's talk on slack about this

> also thinking about this signature makes me wonder if we should fold the defaultTtlSeconds into the config object. I'm leaning "no" on that but just food for thought.
`defaultTtlSeconds` no; the data client deadline, `dataClientOperationTimeoutMilliseconds` should definitely be in the config since network latency affects that (laptop vs prod), and the user should generally not touch that or have to think about it

Reply:
> I think making this required will introduce more friction to getting started and cognitive burden. Let's talk on slack about this

sure.  i asked this question very explicitly in the design review and the outcome was that we said we would make it required.  no one pushed back on it at the time.  but we can discuss if you like.

Reply:
Config required https://github.com/momentohq/client-sdk-dotnet/pull/170/commits/3e36723d10670aa0fc7a791bb94e291cf3166f74